python/tvm/relay/qnn/op/qnn.py

```diff
@@ -219,6 +221,16 @@ def conv2d(data,
     input_zero_point: int
            The zero point of the data distribution.
 
+    input_scale: float
+           The scale for the input tensor. The scale for the input tensor is
+           stored purely for convenience here. See more commentary below.
+
+    kernel_scale: float
```

Comment:
Move these two docstring below `kernel_zero_point` to keep consistent to the order of the parameters.